cardinal/game/archetypes.go

```diff
@@ -2,19 +2,19 @@ package game
 
 import (
 	"github.com/argus-labs/starter-game-template/cardinal/component"
-	"pkg.world.dev/world-engine/cardinal/ecs"
+	"pkg.world.dev/world-engine/cardinal/ecs/component_metadata"
```

Comment:
We are using non-Cardinal API here, I thought we didn't want to do this?

Reply:
We have to. Either this lives in ecs or it lives in it's own package. There's no way around it. 

If we put this in cardinal, then it will cause a circular dependency because ecs uses this interface. 

We could use the previous pattern that you All agreed upon for world context. 

That means creating Another component wrapper. One in cardinal, one in ecs. 

But that would negate the entire component refactor. The whole point was to pull component out of ecs.